backend/src/main/resources/db/migration/internal/V0.116__update_infraction_vessel_size_type.sql

```diff
@@ -0,0 +1,26 @@
+
+UPDATE env_actions t
+SET value = 
+jsonb_set(value, '{infractions}', 
+		  (SELECT jsonb_agg(
+		CASE
+			WHEN j1->'vesselSize' = '"LESS_THAN_12m"'
+			THEN jsonb_set(j1, '{vesselSize}','"11"')
+			WHEN j1->'vesselSize' = '"FROM_12_TO_24m"'
+			THEN jsonb_set(j1, '{vesselSize}','"23"')
+			WHEN j1->'vesselSize' = '"FROM_24_TO_46m"'
+			THEN jsonb_set(j1, '{vesselSize}','"45"')
+			WHEN j1->'vesselSize' = '"MORE_THAN_46m"'
+			THEN jsonb_set(j1, '{vesselSize}','"46"')
+			ELSE j1
+		END
+        )
+     FROM env_actions, jsonb_array_elements(t.value->'infractions') j1
```

Comment:
J'ai l'impression que l'on n'a pas besoin de la jointure avec env_actions
```suggestion
     FROM jsonb_array_elements(t.value->'infractions') j1
```